sig-network/annual-report-2021.md

```diff
@@ -0,0 +1,65 @@
+  - Last community meeting was long enough ago that it is no longer in the minutes history (so, clearly we're due).
+
+# Membership
+
+- Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?
+  - Yes
+- How do you measure membership? By mailing list members, OWNERs, or something else?
+  - We don't currently have an explicit way of measuring membership.
+- How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?
+  - We don't currently have an explicit way of measuring reviewer / approver bandwidth. Everyone feels overburdened, we're not doing anything about it.
+- Is there a healthy onboarding and growth path for contributors in your SIG? What are some activities that the group does to encourage this? What programs are you participating in to grow contributors throughout the contributor ladder?
+  - The easiest way for new contribs to get involved is to attend a meeting and volunteer to help triage an issue during our triage session.
+- What programs do you participate in for new contributors?
+  - No programs specifically targeting new contribs at the moment.
+- Does the group have contributors from multiple companies/affiliations? Can end users/companies contribute in some way that they currently are not?
+  - Yes - we have long-term as well as one-time contributors from a variety of companies. 
+
+# Current initiatives and project health
+
+> Please include links to KEPs and other supporting information that will be beneficial to multiple types of community members. 
+
+- What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? Currently underway? What are some of the longer tail projects that your group is working on?
+- Year to date KEP work review: What’s now stable? Beta? Alpha? Road to alpha?
+  - EndpointSlice: [GA](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/0752-endpointslices)
+  - Dual-stack support: [beta](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/563-dual-stack)
+  - ClusterNetworkPolicy: [road to alpha](https://github.com/kubernetes/enhancements/pull/2522)
+  - Multiple CIDRs: [road to alpha](https://github.com/kubernetes/enhancements/pull/2594)
+  - All-port services: [road to alpha](https://github.com/kubernetes/enhancements/pull/2611/)
+  - Topology-aware-hints: [Alpha](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/2433-topology-aware-hints)
+  - Ingress: [GA](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/1453-ingress-api)
+- What areas and/or subprojects does the group need the most help with?
+  - Tracking project health, membership, progress.
+  - PR reviews
+  - Reporting to other SIGs / community meeting.
+- What's the average open days of a PR and Issue in your group? / what metrics does your group care about and/or measure?
+  - I don't think we currently track this.
```

Comment:
Not necessarily something you need to add, but a few useful dashboards:
- [PR Workloads per SIG](https://k8s.devstats.cncf.io/d/34/pr-workload-per-sig-table?orgId=1&var-period_name=Last%20year)
- [PR Time to Approve](https://k8s.devstats.cncf.io/d/44/pr-time-to-approve-and-merge?var-period=w&var-repogroup_name=SIG%20Network&var-repo_name=kubernetes%2Fkubernetes&var-apichange=All&var-size_name=All&var-kind_name=All) (This is not very accurate, but can be an indicator)